core/src/main/java/org/springframework/security/authentication/UsernamePasswordAuthenticationToken.java

```diff
@@ -47,7 +49,7 @@
 	 * will return <code>false</code>.
 	 *
 	 */
-	public UsernamePasswordAuthenticationToken(Object principal, Object credentials) {
+	protected UsernamePasswordAuthenticationToken(Object principal, Object credentials) {
```

Comment:
We cannot reduce the scope of a public constructor as this is a breaking change. Please leave as-is.